lib/widgets/profile.dart

```diff
@@ -0,0 +1,334 @@
+                ProfileEmailField(user: user),
+                ProfileRoleField(role: user.role),
+                // TODO(#197) render user status
+                // Text("Active about XXX"), // TODO(#196) render active status
+                LocalTimeField(user: user),
+                ProfileDataTable(user: user),
+                const SizedBox(height: 16),
+                FilledButton.icon(
+                  onPressed: () => _createDmNavigation(context),
+                  icon: const Icon(Icons.email),
+                  label: const Text("Send direct message")),
+              ])))));
+  }
+}
+
+class ProfileEmailField extends StatelessWidget {
+  final User user;
+
+  const ProfileEmailField({super.key, required this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    if (user.deliveryEmailStaleDoNotUse == null) {
+      return const SizedBox.shrink();
+    }
+    return Text(user.deliveryEmailStaleDoNotUse!);
+  }
+}
+
+class ProfileRoleField extends StatelessWidget {
+  final UserRole role;
+
+  const ProfileRoleField({super.key, required this.role});
+
+  @override
+  Widget build(BuildContext context) {
+    final roleLabel = switch (role) {
+      UserRole.owner => "Owner",
+      UserRole.administrator => "Administrator",
+      UserRole.moderator => "Moderator",
+      UserRole.member => "Member",
+      UserRole.guest => "Guest",
+      UserRole.unknown => "Unknown",
+    };
+    return Text(roleLabel);
+  }
+}
+
+class LocalTimeField extends StatelessWidget {
+  final User user;
+
+  const LocalTimeField({super.key, required this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    if (user.isBot) {
+      return const SizedBox.shrink();
+    }
+    final userLocalNow = getNowInTimezone(user.timezone);
+    if (userLocalNow == null) {
+      return const SizedBox.shrink();
+    }
+    return Text("${DateFormat.jm().format(userLocalNow)} Local time",
+      style: const TextStyle(fontWeight: FontWeight.normal));
+  }
+}
+
+class ProfileDataTable extends StatelessWidget {
+  final User user;
+
+  const ProfileDataTable({super.key, required this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    if (user.profileData == null) {
+      return const SizedBox.shrink();
+    }
+    final store = PerAccountStoreWidget.of(context);
+    final displayEntries = sortAndAnnotateUserFields(user.profileData!, store.customProfileFields);
+    if (displayEntries.isEmpty) {
+      return const SizedBox.shrink();
+    }
+    return DefaultTextStyle.merge(
+      style: const TextStyle(fontSize: 16, height: 1),
+      textAlign: TextAlign.left,
+      child: Column(
+        children: [
+          const SizedBox(height: 16),
+          Table(
+            border: TableBorder.all(style: BorderStyle.none),
+            columnWidths: const {
+              0: FixedColumnWidth(116),
+              1: FlexColumnWidth()},
+            defaultVerticalAlignment: TableCellVerticalAlignment.top,
```

Comment:
How does `TableCellVerticalAlignment.baseline` behave when one cell has multiple lines of text? If we can get basically top alignment, but with the baselines of the first lines instead of the top edges, that seems ideal.

Reply:
Using `TableCellVerticalAlignment.baseline` with `TextBaseline.alphabetic` still maintains top alignment, so the ideal condition has been achieved!